### ISSUE TITLE
after password change getting "Failed to add account. No Azure tenants"

### ISSUE THREAD

**Issue by rdbossjr** [microsoft/azuredatastudio#9441]
Issue Type: <b>Bug</b>

after password change getting "Failed to add account. No Azure tenants". All of my connections are no longer usable I can't recreate connections after removing them. Have valid Azure login. Can access devOps but not the Azure Database instances. I can access them via SSMS but not in Azure Data Studio.

Azure Data Studio version: azuredatastudio 1.15.1 (83544eba2504f564d785534cc780d2073feec2cc, 2020-02-15T05:03:16.545Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.85GB (4.84GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
managed-instance-dashboard|Mic|0.3.9

</details>

**Comment by rdbossjr** [comment]
I have tried to uninstall Azure Data Studio and reinstall but it looks like all my previous settings moved forward. Is there a way to remove all of the local remnants after an uninstall and start from scratch?

**Comment by Charles-Gagnon** [comment]
@rdbossjr After uninstalling delete the %APPDATA%\azuredatastudio folder and that should remove all settings.

**Comment by kunal-mandalia** [comment]
I experienced something very similar on MacOS. I had to delete the account cache to be able to reconnect. The file was in:

`$HOME/Library/Application Support/azuredatastudio/Azure Accounts/azureTokenCache-azurePublicCloud`

**Comment by Charles-Gagnon** [comment]
Yeah, you can try that first too. For that we have a command you can use to clear the Token Cache - **Azure Accounts: Clear Azure Account Token Cache**

![image](https://user-images.githubusercontent.com/28519865/76010171-55404680-5ec7-11ea-8423-6cd95eef39b5.png)

**Comment by kunal-mandalia** [comment]
Thanks for the quick reply @Charles-Gagnon

**Comment by rdbossjr** [comment]
I removed the azuredatastudio folder and then was able to recreate my connections! Thank you very much for your help!
